modules/core/fsx-lustre/README.md

```diff
@@ -0,0 +1,66 @@
+
+#### Optional
+- `data_bucket_name`: The S3 bucket used for mapping to and from the FSx filesystem
+- `export_path`: Available with `Scratch` and `Persistent_1` deployment types. Specifies the path in the Amazon S3 bucket where the root of your Amazon FSx file system is exported. The path will use the same Amazon S3 bucket as specified in `data_bucket_name`. You can provide an optional prefix to which new and changed data is to be exported from your Amazon FSx for Lustre file system. If an ExportPath value is not provided, Amazon FSx sets a default export path, s3://import-bucket/FSxLustre[creation-timestamp] . The timestamp is in UTC format, for example s3://import-bucket/FSxLustre20181105T222312Z . The Amazon S3 export bucket must be the same as the import bucket specified by ImportPath . If you specify only a bucket name, such as s3://import-bucket, you get a 1:1 mapping of file system objects to S3 bucket objects. This mapping means that the input data in S3 is overwritten on export. If you provide a custom prefix in the export path, such as s3://import-bucket/[custom-optional-prefix] , Amazon FSx exports the contents of your file system to that export prefix in the Amazon S3 bucket.
+  - This parameter is not supported for file systems using the `Persistent_2` deployment type.
+- `import_path`: The path to the Amazon S3 bucket that you’re using as the data repository for your Amazon FSx for Lustre file system. The root of your FSx for Lustre file system will be mapped to the root of the Amazon S3 bucket you select. If you specify a prefix for the Amazon S3 bucket name, only object keys with that prefix are loaded into the file system.
+  - This parameter is not supported for Lustre file systems using the Persistent_2 deployment type.
+- `storage_throughput`: Required with `PERSISTENT_1` and `PERSISTENT_2` deployment types, provisions the amount of read and write throughput for each 1 tebibyte (TiB) of file system storage capacity, in MB/s/TiB. File system throughput capacity is calculated by multiplying ﬁle system storage capacity (TiB) by the PerUnitStorageThroughput (MB/s/TiB). For a 2.4-TiB ﬁle system, provisioning 50 MB/s/TiB of PerUnitStorageThroughput yields 120 MB/s of ﬁle system throughput. You pay for the amount of throughput that you provision.
+  - For `PERSISTENT_1` SSD storage: 50, 100, 200 MB/s/TiB.
+  - For `PERSISTENT_1` HDD storage: 12, 40 MB/s/TiB.
+  - For `PERSISTENT_2` SSD storage: 125, 250, 500, 1000 MB/s/TiB.
+
+#### Input Example
+module manifest example:
+
+```yaml
+name: storage
+path: modules/core/fsx-lustre/
+parameters:
+  - name: private_subnet_ids
+    value: [subnet-abc1234]
```

Comment:
Does this stipulate that the input should be in `[ ]` rather than yaml format?  If so, should it be a comma-separated list?  

Should we think about an example (say down below in the doc) that also shows how to reference the VPC module:
```
Would it be better to show an example like this:

  - name: vpc-id
    valueFrom:
      moduleMetadata:
        group: optionals
        name: networking
        key: VpcId
  - name: private-subnet-ids
    valueFrom:
      moduleMetadata:
        group: optionals
        name: networking
        key: PrivateSubnetIds
```

Reply:
The [] is yaml, it's an alternative style called 'flow style', and the list would be comma separated; i had put only one item in there because lustre only supports one subnet_id anyway. but i will need to test the deployment with valueFrom vpc. I will add a valueFrom example as well